guides/installation_guides/java/mac/README.md

```diff
@@ -1,6 +1,10 @@
-# Cosmos Guides
-> Your personal library of every algorithm and data structure code that you will ever encounter
+# Java Installation Guide
+### Download
+Download the .dmg from the [Java website](https://www.java.com/en/download/)
```

Comment:
But backticks around the words .dmg in all the places. This will make it clearer to read.